core/ledger/kvledger/txmgmt/statedb/statecouchdb/statecouchdb_test_export.go

```diff
@@ -13,61 +13,76 @@ import (
 	"time"
 
 	"github.com/hyperledger/fabric/common/metrics/disabled"
+	"github.com/hyperledger/fabric/core/ledger"
 	"github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/statedb"
-	"github.com/hyperledger/fabric/core/ledger/util/couchdb"
-	"github.com/hyperledger/fabric/core/ledger/util/couchdbtest"
+	"github.com/hyperledger/fabric/integration/runner"
 	"github.com/stretchr/testify/require"
 )
 
 // testVDBEnv provides a couch db backed versioned db for testing
 type testVDBEnv struct {
-	t              testing.TB
+	t              *testing.T
 	couchAddress   string
 	DBProvider     statedb.VersionedDBProvider
-	config         *couchdb.Config
+	config         *ledger.CouchDBConfig
 	cleanupCouchDB func()
 	cache          *cache
 	sysNamespaces  []string
 }
 
-func (env *testVDBEnv) init(t testing.TB, sysNamespaces []string) {
+func (env *testVDBEnv) init(t *testing.T, sysNamespaces []string, createdVersionedDB bool) {
```

Comment:
I find this change `createdVersionedDB` a hacky. Not sure what it means to have a `testVDBEnv` without `VDB`. 
If you want to use only couchdb container without constructing versiondb, I would prefer to introduce another struct for `couchcontainer` and create a single instance of that - which is used within `testVDBEnv` and invoked directly from other tests. Being explicitly makes it more readable.